Sources/FioriAR/ARCards/Views/Authoring/SceneAuthoringController.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  SceneAuthoringController.swift
+//
+//
+//  Created by O'Brien, Patrick on 12/16/21.
+//
+
+import SAPFoundation
+import SwiftUI
+
+/**
+ Provides the flow for authoring an AR Annotation Scene using UIKit. Used to create the content for the cards, select an anchor Image, and position the entities in their real world locations. Publishing the scene using Mobile Services with an SAPURLSession.
+ The onSceneEdit modifier provides a callback on editing events. When a scene was published, i.e. created or updated in SAP Mobile Services, then `.published(sceneID)` is called and returns the technical id of the scene.
+ Note: Push onto a navigation stack, modal presenting not supported
+
+ ## Usage
+ ```
+ @objc func launchSceneAuthoring(sender: UIButton) {
+     let sceneAuthoringController = SceneAuthoringController(title: "Annotations",
+                                                             serviceURL:  URL(string: IntegrationTest.System.redirectURL)!,
+                                                             sapURLSession: sapURLSession,
+                                                             sceneIdentifier: SceneIdentifyingAttribute.id(IntegrationTest.TestData.sceneId))
+     self.navigationController?.pushViewController(sceneAuthoringController, animated: true)
+ }
+ ```
+ */
+public class SceneAuthoringController: UIHostingController<SceneAuthoringView> {
+    /// Initializer
+    /// - Parameters:
+    ///   - title: Title of the Scene
+    ///   - serviceURL: SAPURLSession to provide credentials to Mobile Services
+    ///   - sapURLSession: If nil then a new scene is created.
```

Comment:
Incorrect parameter description

Reply:
Ah good catch. I reused the parameter description from `SceneAuthoringView`'s [_initializer_](https://github.com/SAP/cloud-sdk-ios-fiori-ar/blob/4dd447cbf07ca57f1b5d578bb5c98aa28aa374b2/Sources/FioriAR/ARCards/Views/Authoring/SceneAuthoringView.swift#L55) and have made a fix for that too.